ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -80,7 +81,10 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
                     _shape_matrices[ip]);
         }
 
-        _hydraulic_conductivity = hydraulic_conductivity;
+        _hydraulic_conductivity = [&hydraulic_conductivity, &e]()
```

Comment:
why not simply store an evaluated value here instead of a function? the conductivity is a function of only an element.